README.md

```diff
@@ -4,16 +4,18 @@
     <img src="docs/logo.png" width="300">
 </p>
 
-This non-profit repository intention is to help the community to learn the DevOps methodology and culture throughout a series of classes and hands-on laboratories and become ready to start working as Automation/Cloud/DevOps Engineer.
+This repository's intention is to help the community in learning the DevOps methodologies and culture throughout a series of classes and hands-on exercises. The content of the classes that will be presented has been designed in a way to prepare you for working as an Automation/Cloud/DevOps Engineer.
 
-Feel free to contribute with content (PR), help peer reviewing code or just going through the exercises!
+These classes start with explaining different concepts at a high level and then goes into some detail. Some of the main topics currently being covered can be seen [here](#content).
 
-If you want to make part of the team behind this, get in touch on help@devopsacademy.com.au.
+There will also be some new content coming in the future, more details can be found at the bottom of the file in the [Future Work](#future-work) section.
```

Comment:
I don't really like the idea of future work as that can get against us in case we don't deliver.
We are tracking new content as issues and adding cards to the project boards

Reply:
@denstorti @kikobr82  what do you guys think?

Reply:
I don't like it as well. we should keep those as issue and work as we have opportunity.

Reply:
ok, i guess the majority prefers not to have that section which is fine, i'll can remove it.  I just thought of it as a good way to give some sort of idea to any potential new students and even existing one about whats in the pipeline and what kind of content to might be available in the future.

Reply:
start adding your ideas as cards to the project then and remove the future work section

Reply:
That's been updated and i raised a few issues instead